Blockzilla/IntroViewController.swift

```diff
@@ -111,7 +111,7 @@ class IntroViewController: UIViewController {
 
 extension IntroViewController: ScrollViewControllerDelegate {
     func scrollViewController(scrollViewController: ScrollViewController, didDismissSlideDeck bool: Bool) {
-        Telemetry.default.recordEvent(category: TelemetryEventCategory.firstRun, method: TelemetryEventMethod.completed, object: TelemetryEventObject.onboarding)
+        Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.click, object: TelemetryEventObject.onboarding, value: "finish")
```

Comment:
If this is a new probe, it's fine to change now since we haven't released it yet.